### PR TITLE
podman: honor --storage-opt again

### PR DESCRIPTION
**Pull request by giuseppe**
possibly a regression after we moved to Cobra

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM. Surprised CI didn't catch this, I thought we used this in our standard CI podman invocation

File: cmd/podman/libpodruntime/runtime.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func GetRuntime(c *cliconfig.PodmanCommand) (*libpod.Runtime, error) {
 	if c.Flags().Changed("storage-driver") {
 		storageOpts.GraphDriverName = c.GlobalFlags.StorageDriver
 	}
-	if c.Flags().Changed("storage-opt") {
+	if len(c.GlobalFlags.StorageOpts) > 0 {
```
Comment (TomSweeneyRedHat) [inline cmd/podman/libpodruntime/runtime.go]: Interesting.  I thiought the "Changed" would have handled this.  @baude should we elook at the Changed commands use elsewhere?

Comment (rhatdan) [reply cmd/podman/libpodruntime/runtime.go]: Changed means that the user specified it, correct?

Comment (TomSweeneyRedHat) [reply cmd/podman/libpodruntime/runtime.go]: That was my understanding for Changed.

**Pull request merged**